### DISCUSSION TITLE
How to know if someone is already working on an issue

### DISCUSSION THREAD

**Discussion by anon-oss** [o3de/o3de#16064] [General]
I am new to O3de and have some doubts about how its development community work. If an issue does not have assignee does it mean that no one is working to fix it.

For example I was able to fix https://github.com/o3de/o3de/issues/14464 and https://github.com/o3de/o3de/issues/6610 when I was installing O3de on mutiple systems (linux and windows). But to make a pull request out of it, I should make the code better.

**Comment by AMZN-daimini** [comment]
Hello!
Yes, generally you can assume no-one's working on an unassigned issue. Feel free to publish a PR for it when you have the chance, reviewers will try to help you with improving the code if necessary :)

Regarding https://github.com/o3de/o3de/issues/6610, it looks like it may a few more logistical issues than just the technical aspect of it, but feel free to reach out to @o3de/sig-release or @o3de/sig-build to propose a solution.
